typescript/thing-description/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "wot-thing-description-types",
```

Comment:
same as above

I wonder whether the folder name (currently `thing-description` should be aligned with the name)

Reply:
Consistency ?
`wot-typescript-definitions`     (exists already)
vs.
`wot-typescript-thing-description`
vs.
`wot-thing-description-types`

Comments?

Reply:
Scripting Call 2021-08-09
decided to keep "thing-description" folder name